src/wp-admin/includes/post.php

```diff
@@ -778,16 +778,18 @@ function get_default_post_to_edit( $post_type = 'post', $create_in_db = false )
  * @param string $date    Optional. Post date.
  * @param string $type    Optional. Post type.
  * @param string $status  Optional. Post status.
+ * @param integer $author Optional. Post author ID.
  * @return int Post ID if post exists, 0 otherwise.
  */
-function post_exists( $title, $content = '', $date = '', $type = '', $status = '' ) {
+function post_exists( $title, $content = '', $date = '', $type = '', $status = '', $author = '' ) {
```

Comment:
```suggestion
function post_exists( $title, $content = '', $date = '', $type = '', $status = '', $author = 0 ) {
```

The `$author` parameter is an integer, so we should default to a value of that type.